libs/sr/solver/src/childWorker.ts

```diff
@@ -123,7 +123,7 @@ async function start() {
                 sphere.stats,
                 rope.stats,
               ])
-
+              let skipped = 0
```

Comment:
Isn't this just getting reset every loop iteration?

Reply:
You are right, my bad.